hastory-data/src/Data/Hastory/Utils.hs

```diff
@@ -20,3 +31,19 @@ doCountsWith conv func = foldl go HM.empty
       where
         a Nothing = Just 1
         a (Just d) = Just $ d + func k
+
+dataBaseSpec :: SpecWith SqlBackend -> Spec
+dataBaseSpec = around withDatabase
+
+withDatabase :: ActionWith SqlBackend -> IO ()
+withDatabase = runNoLoggingT . withSqlConn setupDb . fmap liftIO
```

Comment:
Nice, but you still need to run the migrations I think.

Reply:
The migration setup for the DB is being done in `setupDb`, which runs the migrations using [runMigrationSilne](https://www.stackage.org/haddock/lts-11.6/persistent-2.8.2/Database-Persist-Sql.html#v:runMigrationSilent).